src/gui/mainwindow.cpp

```diff
@@ -101,13 +101,18 @@ void qt_mac_set_dock_menu(QMenu *menu);
 
 namespace
 {
-#define SETTINGS_KEY(name) "MainWindow/" name
+#define SETTINGS_KEY(name) "GUI/" name
 
     // ExecutionLog properties keys
-#define EXECUTIONLOG_SETTINGS_KEY(name) SETTINGS_KEY("ExecutionLog/") name
+#define EXECUTIONLOG_SETTINGS_KEY(name) SETTINGS_KEY("Log/") name
     const QString KEY_EXECUTIONLOG_ENABLED = EXECUTIONLOG_SETTINGS_KEY("Enabled");
     const QString KEY_EXECUTIONLOG_TYPES = EXECUTIONLOG_SETTINGS_KEY("Types");
 
+    // Notifications properties keys
+#define NOTIFICATIONS_SETTINGS_KEY(name) SETTINGS_KEY("Notifications/") name
```

Comment:
I don't have an example but my gut tells me that "Notifications" is the expected name for the setting.